contributors/guide/scalability-good-practices.md

```diff
@@ -19,31 +19,31 @@
 ## Who should read this document and what is in it?
 This document is targeted at developers of "vanilla Kubernetes" who do not want their changes rolled-back or blocked because they cause performance regressions. It contains some of the knowledge and experience gathered by the scalability team over more than two years.
  
-It is presented as a set of past examples, as realistic as possible, which broke scalability tests. Those are followed by explanations and general suggestions on how to avoid causing similar problems.
+It is presented as a set of real, or close-to-real examples from the past which broke scalability tests, followed by some explanations and general suggestions on how to avoid causing similar problems.
```

Comment:
Let's just get rid of the "real" part. It's a red herring. :)

Reply:
Done. (I like herring. Great pickled herring in Poland.)